passes/unclosetx/testdata/src/a/a.go

```diff
@@ -73,3 +73,22 @@ func f6(ctx context.Context, client *spanner.Client) error {
 	}
 	return nil
 }
+
+func f7(ctx context.Context, client *spanner.Client) error {
```

Comment:
Could you add a test case which does not call `Close` method with a `// want` comment.

Reply:
`f5` is that the testcase.
https://github.com/gcpug/zagane/blob/effcadc580dfe1f2250d1ce1c7bb234845b82950/passes/unclosetx/testdata/src/a/a.go#L54-L57